File: Qipai4Core/src/net/wdqipai/core/model/IChairModel.java
package net.wdqipai.core.model;

import net.wdqipai.core.*;

public interface IChairModel
{
	//int getId();

	int getId();

        /**
        *
        * @return
        */
	IUserModel getUser();

	void setUser(IUserModel user);

	boolean isReady();

	void setReady(boolean value);

	/** 
	 ready的附加信息
	 
	 @param value
	*/
	void setReadyAdd(String value);

	String getReadyAdd();

	void reset();

	String toXMLString();

	//string ContentXml { get; }


}

File: CSharpServer/Qipai4Core/Qipai4Core/net/silverfoxserver/core/model/IChairModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace net.silverfoxserver.core.model
{
    public interface IChairModel
    {
        //int getId();

        int Id { get; }

        IUserModel getUser();

        IUserModel User{ get; }

        void setUser(IUserModel user);

        bool isReady{ get;}

        void setReady(bool value);

        /// <summary>
        /// ready的附加信息
        /// </summary>
        /// <param name="value"></param>
        void setReadyAdd(string value);

        string getReadyAdd();

        void reset();

        string toXMLString();

        //string ContentXml { get; }
        

    }
}
